script.js

```diff
@@ -0,0 +1,43 @@
+/* HW_1
+
+let country = `Moldova`; 
```

Comment:
I’m not sure if it has difference but while declaring a string variable I use regular quotes ("") or single quotes (‘’) rather than backticks

Reply:
Not sure either. Will check, thank you!